#name : try, catch
# --
try {
    $0
} catch (${1:Exception} ${2:ex}) {
} finally {
}
